pymc/math.py

```diff
@@ -577,4 +528,8 @@ def block_diagonal(matrices, sparse=False, format="csr"):
     """
     if len(matrices) == 1:  # graph optimization
         return matrices[0]
-    return BlockDiagonalMatrix(sparse=sparse, format=format)(*matrices)
+
```

Comment:
Can you add a FutureWarning mentioning this is deprecated in favor of the pytensor function, so we can remove this thin wrapper sometime in the future?